app/controllers/checkouts_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CheckoutsController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @checkout_session = Payment.pay(current_user, 'stripe', params[:payment_method_token], 'payment', 'price_1Jrc1cCStSxJzrVqqCSRgtnR', params[:id], success_checkouts_url, root_url)
```

Comment:
@checkout_session = Payment.pay(current_user, payment_params)
in service: include Rails.application.routes.url_helpers

def payment_params
  params.permit(:id, :payment_method_token)
end

Reply:
change service name to `PaymentService`